peg/Parser.hx

```diff
@@ -369,6 +435,10 @@ class Parser {
 						case _: throw new UnexpectedTokenException(token);
 					}
 					v.type = type;
+					var t = parseDocTagParamType(fn, v.name);
+					if (t != null && v.type == TMixed) {
```

Comment:
Better check `v.type` before executing `parseDocTagParamType`

Reply:
done in 09f1f4c